crates/torii/core/src/processors/event_message.rs

```diff
@@ -73,8 +73,7 @@ where
         entity.deserialize(&mut keys_and_unpacked)?;
 
         // TODO: this must come from some torii's configuration.
-        let historical =
-            config.historical_events.contains(&format!("{}-{}", model.namespace, model.name));
+        let historical = config.is_historical(&format!("{}-{}", model.namespace, model.name));
```

Comment:
We should use `dojo::utils` here to compute the tag from the namespace and name to have a more reliable way to actually compute the tag.